app/src/userpages/components/StreamPage/Edit/PreviewView.jsx

```diff
@@ -59,31 +60,32 @@ const LOCAL_DATA_LIST_LENGTH = 20
 
 const initialState = Array(PREVIEW_TABLE_LENGTH).fill(undefined)
 
+const areMessagesSame = (a, b) => {
```

Comment:
This should fix https://linear.app/streamr/issue/FRONT-355/make-the-stream-inspector-filters-out-dupped-messages